code-scanning/clj-holmes.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: clj-holmes
+
+on:
+  push:
+    branches: [ $default-branch, $protected-branches ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ $default-branch ]
+  schedule:
+    - cron: $cron-weekly
+
+jobs:
+  clj-holmes:
+    name: Run clj-holmes scanning
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
```

Comment:
What's this actions permission needed for...?

Reply:
just following the other workflows adding the same permissions.

I think only read permission is needed. 
Based on your comments, what I should do:
- remove overwrite permissions section at the action level.
- keep read permission at the job level.

Can you pls confirm?

Reply:
After some test runs, I found at least 'security-events' write permission is required, otherwise action `github/codeql-action/upload-sarif@v1` will fail. 

```
    permissions:
      security-events: write
```

What I should do is:
- Remove permissions at the action level.
- Add the required permissions at the job level

@nickfyson can you pls confirm? Thanks

Reply:
It was just the `actions` permission I was querying, but it sounds like you agree that isn't needed?

Similarly to most other workflows, I would expect the required permissions to be just...

```
permissions:
   contents: read
   security-events: write
```